.buildkite/scripts/steps/test/scout_configs.sh

```diff
@@ -119,7 +127,10 @@ fi
 if [[ ${#failedConfigs[@]} -gt 0 ]]; then
   echo "❌ Failed tests:"
   printf '%s\n' "${failedConfigs[@]}"
-  buildkite-agent meta-data set "$FAILED_CONFIGS_KEY" "$failedConfigs"
+
+  # Store plain config paths for retry (convert array to newline-separated string)
+  failed_configs_for_retry=$(printf '%s\n' "${failedConfigPaths[@]}")
+  buildkite-agent meta-data set "$FAILED_CONFIGS_KEY" "$failed_configs_for_retry"
```

Comment:
Should we make it clear that these are Scout configs to avoid ambiguity? 

```suggestion
  buildkite-agent meta-data set "$FAILED_SCOUT_CONFIGS_KEY" "$failed_configs_for_retry"
```